app/services/audit/checksum.rb

```diff
@@ -34,6 +34,8 @@ def self.validate_list_of_druids(druid_list_file_path)
 
     # validate objects with a particular status on a particular moab_storage_root
     def self.validate_status_root(status, storage_root_name)
+      raise ArgumentError, "invalid status #{status}" unless CompleteMoab.statuses.key?(status)
```

Comment:
AR used to object to querying on an invalid enum value. It no longer does, so explicitly checking and raising.

Reply:
🥇